sockeye/encoder.py

```diff
@@ -392,11 +354,19 @@ def encode(self,
         :param seq_len: sequence length.
         :return: (batch_size, source_seq_len, num_embed)
         """
-        embedding = mx.sym.broadcast_add(data,
-                                         mx.sym.BlockGrad(mx.symbol.Custom(length=seq_len,
-                                                                           depth=self.num_embed,
-                                                                           name="%spositional_encodings" % self.prefix,
-                                                                           op_type='positional_encodings')))
+        # add positional embeddings to data
+        if self.scale_input:
```

Comment:
would it make sense to call the parameters `scale_up_init` and `scale_down_positions` to make it a bit clearer why one is `**0.5` while the other is `**-0.5`?

Reply:
makes sense, will change